include/mbedtls/check_config.h

```diff
@@ -293,6 +293,10 @@
 #error "!MBEDTLS_SSL_KEEP_PEER_CERTIFICATE requires MBEDTLS_SHA512_C, MBEDTLS_SHA256_C or MBEDTLS_SHA1_C"
 #endif
 
+#if defined(MBEDTLS_CERTS_C) && !defined(MBEDTLS_PK_C)
```

Comment:
Should `MBEDTLS_CERTS_C` require `MBEDTLS_X509_CRT_PARSE_C`, and `MBEDTLS_X509_CRT_PARSE_C` (and other X509 modules) require `MBEDTLS_PK_C`?

Reply:
That's a fair suggestion. I've changed the dependency to now be on `MBEDTLS_X509_USE_C`, as the X.509 self-test that uses `MBEDTLS_CERTS_C` only depends on `MBEDTLS_X509_USE_C` not `MBEDTLS_X509_CRT_PARSE_C`.